docs/src/pages/guides/right-to-left/right-to-left.md

```diff
@@ -125,18 +125,38 @@ _Use the direction toggle button on the top right corner to flip the whole docum
 
 ## Opting out of rtl transformation
 
-### JSS
+### emotion & styled-components
 
-If you want to prevent a specific rule-set from being affected by the `rtl` transformation you can add `flip: false` at the beginning.
+You have to use the template literal syntax and add the `/* @noflip */` directive before the rule or property for which you want to disable right-to-left styles.
 
-_Use the direction toggle button on the top right corner to see the effect._
+```jsx
+const AffectedText = styled('div')`
+  text-align: left;
+`;
+
+const UnaffectedText = styled('div')`
+  /* @noflip */
+  text-align: left;
+`;
+```
 
-{{"demo": "pages/guides/right-to-left/RtlOptOut.js", "hideEditButton": true}}
+{{"demo": "pages/guides/right-to-left/RtlOptOutStylis.js", "hideToolbar": true}}
```

Comment:
One detail, if we only want to disable the preview and keep the toolbar, we could also use `"defaultCodeOpen": false`.

Reply:
Most of this demo is irrelevant. It really is just about `noflip` usage.